src/index.js

```diff
@@ -504,46 +504,46 @@
                         .on('paused', err => {
                             if (err) {
                                 vm.$emit('pouchdb-changes-error', {
-                                    db: localDB,
+                                    db: db,
                                     error: err,
                                 });
                                 return;
                             }
                             numPaused += 1;
                             if (numPaused >= 2) {
                                 vm.$emit('pouchdb-changes-paused', {
-                                    db: localDB,
+                                    db: db,
                                     paused: true,
                                 });
                             }
                         })
                         .on('change', info => {
                             vm.$emit('pouchdb-changes-change', {
-                                db: localDB,
+                                db: db,
```

Comment:
Not sure where db is supposed to come from since you only have localDB and remoteDB

Also in lines: 528 534 540 546

Reply:
db is what you have, there's no localDB and remoteDB parameter for the changes method, unlike with the sync, push, and pull methods. I just added another bug fix since there's no changes() method on db. The changes() method needs to be called on databases[db].